src/client/components/MapboxglMap.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div
```

Comment:
Use the same name for the component file and the root element, so `mapbox-map`. I think the `gl` bit is an implementation detail.

Reply:
will change this